cmd/cluster/checkbanneduser.go

```diff
@@ -0,0 +1,69 @@
+package cluster
+
+import (
+	"fmt"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+const BanCodeExportControlCompliance = "export_control_compliance"
+
+func newCmdCheckBannedUser() *cobra.Command {
+	return &cobra.Command{
+		Use:               "check-banned-user",
+		Short:             "Checks if the cluster owner is a banned user.",
```

Comment:
We should add documentation like https://github.com/openshift/osdctl/blob/master/cmd/network/verification.go#L57-L83

Reply:
To loop in our slack conversation - I've added `[CLUSTER_ID]` to the usage here.